tools/autotag/util/mivisionx.py

```diff
@@ -34,7 +34,7 @@ def mivisionx_processor(data: ReleaseLib, template: str, _) -> bool:
 </p>
 
 ## Online Documentation
-[MIVisionX Documentation](https://gpuopen-professionalcompute-libraries.github.io/MIVisionX)
+[MIVisionX Documentation](https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html)
```

Comment:
```suggestion
[MIVisionX Documentation](https://rocm.github.io/MIVisionX/)
```
This morning, I noticed MIVisionX has a github pages version of their docs, which seems to be quite up-to-date (as of 3 weeks ago).  However, it doesn't follow the rocm.docs.amd.com template style.  Should we refer to it instead of the doxygen version?

Reply:
I think the better approach is to sync the docs so the website builds using the updated GitHub content